crates/ruff_linter/src/rules/flake8_pytest_style/rules/fixture.rs

```diff
@@ -693,15 +710,22 @@ fn check_fixture_decorator(checker: &Checker, func_name: &str, decorator: &Decor
         Expr::Call(ast::ExprCall {
             func,
             arguments,
-            range: _,
+            range,
             node_index: _,
         }) => {
             if checker.enabled(Rule::PytestFixtureIncorrectParenthesesStyle) {
                 if !checker.settings.flake8_pytest_style.fixture_parentheses
                     && arguments.args.is_empty()
                     && arguments.keywords.is_empty()
                 {
-                    let fix = Fix::safe_edit(Edit::deletion(func.end(), decorator.end()));
+                    let fix = Fix::applicable_edit(
+                        Edit::deletion(func.end(), decorator.end()),
+                        if checker.comment_ranges().intersects(*range) {
```

Comment:
Could we limit the range to `func.end()..decorator.end()`?

Reply:
Yep